search_shards.go

```diff
@@ -0,0 +1,153 @@
+// Copyright 2012-present Oliver Eilhard. All rights reserved.
+// Use of this source code is governed by a MIT-license.
+// See http://olivere.mit-license.org/license.txt for details.
+
+package elastic
+
+import (
+	"context"
+	"fmt"
+	"github.com/olivere/elastic/uritemplates"
```

Comment:
Can you put that into a separate block, just like in the other services, please?